recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
```

Comment:
Ok sounds good. Just as a reminder though, to update a version once a feedstock has been created (i.e the staged-recipe PR has been merged), you would put in a pull request on the feedstock repo to update the recipe.